src/cargo/ops/cargo_new.rs

```diff
@@ -22,6 +26,8 @@ pub struct NewOptions<'a> {
     pub bin: bool,
     pub path: &'a str,
     pub name: Option<&'a str>,
+    pub license: Option<Vec<License>>,
```

Comment:
This may be best as `&'a [License]` as it's what most of the rest of the struct looks like